### PR TITLE
Clarifying gradient value

### PR DESCRIPTION
**Pull request by b-wils**

### CODE REVIEW

**Review by mbasaglia — APPROVED** [review]

**Comment by mbasaglia** [comment]
Looks like a minor change so we could put it on 1.0.1

**Comment by mbasaglia** [comment]
@cla-bot check

**Review by mgrudzinska — APPROVED** [review]

**Comment by mbasaglia** [comment]
Before merging we should decide whether to put it in 1.0.1, as that would change the target branch

**Pull request merged**
